src/oge/reference_tables/epa_eia_crosswalk_manual.csv

```diff
@@ -157,3 +157,37 @@ plant_id_epa,emissions_unit_id_epa,plant_id_eia,generator_id,start_year,end_year
 1571,SMECO,65285,GT1,2021,2022,This generator switches plant IDs for two years then switches back to its original plant ID
 1571,1,65285,ST1,2021,,This generator switches plant IDs the year it retires (2021)
 1571,2,65285,ST2,2021,,This generator switches plant IDs the year it retires (2021)
+3,8,3,A3C1,2023,,This generator (CEMS unit) had no CEMS unit (generator) assigned
+3,8,3,A3ST,2023,,This generator (CEMS unit) had no CEMS unit (generator) assigned
+47,CCT10,47,GT10,2023,,This generator (CEMS unit) had no CEMS unit (generator) assigned
+47,CCT11,47,GT11,2023,,This generator (CEMS unit) had no CEMS unit (generator) assigned
+47,CCT9,47,GT9,2023,,This generator (CEMS unit) had no CEMS unit (generator) assigned
+56,CC1,56,LEC1,2023,,This generator (CEMS unit) had no CEMS unit (generator) assigned
+56,CC1,56,LEC2,2023,,This generator (CEMS unit) had no CEMS unit (generator) assigned

```

Comment:
How was CC1 matched to both LEC1 and LEC2? Is there anything else we can confirm like the magnitude of reported fuel consumption or generation?

Reply:
EIA has two generators LEC1 and LEC2. The fuel consumed and net electricity generated in 2023 by these generators is as follows:
```
                                                          fuel_consumed_mmbtu  net_generation_mwh
plant_id_eia generator_id energy_source_code report_date                                         
56           LEC1         NG                 2023-01-01                   0.0                 0.0
                                             2023-02-01                   0.0                 0.0
                                             2023-03-01                   0.0                 0.0
                                             2023-04-01                   0.0                 0.0
                                             2023-05-01                   0.0                 0.0
                                             2023-06-01                   0.0                 0.0
                                             2023-07-01                   0.0                 0.0
                                             2023-08-01                   0.0                 0.0
                                             2023-09-01              991829.0             97560.0
                                             2023-10-01             2102061.0            208998.0
                                             2023-11-01             2810252.0            286513.0
                                             2023-12-01             2793002.0            288323.0
             LEC2         NG                 2023-01-01                   0.0                 0.0
                                             2023-02-01                   0.0                 0.0
                                             2023-03-01                   0.0                 0.0
                                             2023-04-01                   0.0                 0.0
                                             2023-05-01                   0.0                 0.0
                                             2023-06-01                   0.0                 0.0
                                             2023-07-01                   0.0                 0.0
                                             2023-08-01                   0.0                 0.0
                                             2023-09-01                   0.0             51482.0
                                             2023-10-01                   0.0            113454.0
                                             2023-11-01                   0.0            147457.0
                                             2023-12-01                   0.0            147555.0
```
LEC2 does generate electricity but does not consume any fuel.  EPA's CAMPD reports only one unit CC1